oclint-scripts/build

```diff
@@ -29,8 +29,6 @@ def build_command(llvm_root, is_release, module_extras, source_path):
     cmd_build = cmake.builder(source_path)
     if is_release:
         cmd_build.release_build()
-    if environment.is_unix():
-        cmd_build.use_clang_compiler(llvm_root)
```

Comment:
I would have done the opposite, using a flag to force local clang. By the way, it is really usefull only when not using system llvm (which most users won't do IMHO). But you are the maintainer, I leave the final decision to you.